geotrek/api/v2/serializers.py

```diff
@@ -1,3 +1,4 @@
+from bs4 import BeautifulSoup
```

Comment:
Maybe you forgot to add `bs4` to dependencies.

Reply:
beautifulsoup4==4.11.1 requirements.txt